### ISSUE TITLE
3 tie elements on 1 note in musicXML

### ISSUE THREAD

**Issue by RensBloom** [Audiveris/audiveris#596]
My exported xml contained 3 ties on a single note after it detected two ties. MusicXML should contain a maximum of 1 stop tie and 1 start tie.
```
      <note default-x="859">
        <pitch>
          <step>F</step>
          <octave>4</octave>
        </pitch>
        <duration>12</duration>
        <tie type="start"></tie>
        <tie type="stop"></tie>
        <tie type="start"></tie>
        <voice>1</voice>
        <type>quarter</type>
        <stem default-y="-20">down</stem>
        <notations>
          <ornaments>
            <trill-mark placement="above"></trill-mark>
          </ornaments>
          <tied type="start" orientation="over"></tied>
          <tied type="stop"></tied>
          <tied type="start" orientation="under"></tied>
        </notations>
      </note>
```

![image](https://user-images.githubusercontent.com/5883286/202230755-6612f43b-5e36-45f0-ae50-6cb476a59041.png)

I put it here in the issue tracker, maybe I'll solve it on a later day (but I don't want to forget this).

**Comment by hbitteur** [comment]
I agree, _at least_ one of these 2 starting ties must be a false positive.
Could you check with the input image? 

Anyway, the OMR engine should check and forbid this. I need to have a look at the code.

**Comment by RensBloom** [comment]
All ties here are false positives, but it doesn't surprise me since a lot of smaller notes are used (in a cadenza) and those are not supported yet. My point was indeed that this should forbidden (either in OMR or in musicXML export).

**Comment by hbitteur** [comment]
Smaller note heads are available, provided you set the proper processing switches on.
Use menu `Book  | Set book parameters` for this.

There are different switches, one for small black, one for small void, one for small whole.
I think I should simplify this and use a single processing switch for all of them.

**Comment by hbitteur** [comment]
No news on this issue.
Should we close it?
